src/low-preview/ci.service.js

```diff
@@ -213,4 +213,49 @@ export default class CIResponsive {
       setBackgroundSrc(imgNode, cloudimageUrl, lazy, src, isSVG, dataSrcAttr);
     }
   }
+
+  setSrc(src, element) {
+    const {imgSelector, bgSelector} = this.config;
+    const isImage = element.hasAttribute(imgSelector);
+    const isBackground = element.hasAttribute(bgSelector);
+
+    let isProcessed;
+
+    if (isImage) {
+      isProcessed = element.classList.contains('ci-image');
+
+      if (isProcessed) {
+        const elementParent = element.parentNode;
+        element.classList.remove('ci-image-loaded');
+        element.classList.remove('lazyloaded');
+        element.classList.remove('ci-image');
+
+        element.setAttribute(imgSelector, src);
+        elementParent.parentNode.replaceChild(element, elementParent);
+      }
+
+      this.getBasicInfo(element, false, false, 'image');
+    }
+
+    if (isBackground) {
+      const lowPreviewDiv = element.children[0];
```

Comment:
const lowPreviewWrapper = element.children[0];
we can do here isProcessed = lowPreviewWrapper && lowPreviewWrapper.classList.contains('ci-image-loaded')